### PR TITLE
chore(dependabot): try to ignore some directories

### PR DESCRIPTION
**Pull request by xxchan**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?
https://github.com/dependabot/dependabot-core/issues/4364#issuecomment-2002406602
https://github.com/risingwavelabs/risingwave/pull/16101#discussion_r1548916995

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added test labels as necessary. See [details](https://github.com/risingwavelabs/risingwave/blob/main/docs/developer-guide.md#ci-labels-guide).
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

**Comment by BugenZhao** [comment]
Why "try try"? 😄

File: .github/dependabot.yml [hunk 1]
```diff
@@ -35,4 +35,17 @@ updates:
...
+        - "google-cloud*"
+# Don't update these directories
+- package-ecosystem: cargo
+  directory: /src/workspace-hack
+  schedule:
+    interval: "daily"
+  ignore:
+    - dependency-name: "*"
+- package-ecosystem: cargo
+  directory: /integration_tests/feature-store
```
Comment (BugenZhao) [inline .github/dependabot.yml:47]: Does it work for subdirectories? `/integration_tests` can meh?

Comment (xxchan) [reply .github/dependabot.yml:47]: idk

**Review by BugenZhao — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (xiangjinwu) [reply .github/dependabot.yml:47]: No we get the answer: no

https://github.com/risingwavelabs/risingwave/network/updates/809611443

**Pull request merged**